vision_datasets/common/data_manifest.py

```diff
@@ -91,7 +91,11 @@ def __init__(self, id, img_path, width, height, labels):
             img_path (str): path to image
             width (int): image width
             height (int): image height
-            labels (list or dict): classification: [c_id] for multiclass, [c_id1, c_id2, ...] for multilabel; detection: [c_id, left, top, right, bottom]; dict[task, labels] for multitask dataset
+            labels (list or dict):
+                classification: [c_id] for multiclass, [c_id1, c_id2, ...] for multilabel;
+                detection: [c_id, left, top, right, bottom];
+                image caption: dict of {'image_id': image_id, 'image_file': filename, 'caption': [dummy strings]};
```

Comment:
Does that mean one image can have multiple captions?

Reply:
Seems not the case in the unit tests

Reply:
Yes, one image can have multiple captions. In the unit tests, each image has 5 captions.

Reply:
got it.